src/main/asciidoc/override/json.adoc

```diff
@@ -116,3 +116,27 @@ You get values from a JSON array using the `getXXX` methods, for example:
 ==== Encoding a JSON array to a String
 
 You use {@link io.vertx.core.json.JsonArray#encode} to encode the array to a String form.
+
+
+=== Json Pointers
```

Comment:
this documentation will only be for Java (override).

I think we need to have a new doc for pointers so it is generated in all languages.